util/java/libtiled-java/src/main/java/org/mapeditor/io/TMXMapReader.java

```diff
@@ -104,7 +129,13 @@
     /**
      * Constructor for TMXMapReader.
      */
-    public TMXMapReader() {
+    public TMXMapReader()
+    {
+        try {
+            unmarshaller = JAXBContext.newInstance(classesToBeBound.toArray(new Class[0])).createUnmarshaller();
+        } catch (JAXBException e) {
```

Comment:
When can this happen and what do we gain from catching this exception?

Reply:
actually it would be better to make constructor throw exception :thinking: